### ISSUE TITLE
Receiving import error

### ISSUE THREAD

**Issue by ProgrammingWithCybineer** [ProgrammingWithCybineer/Python_Shark_Attack_Analysis#8]
ImportError: cannot import name 'DB' from partially initialized module 'database' (most likely due to a circular import) 

File "c:\Users\Cybineer\Desktop\Python_Shark_Attack_Analysis\createaccount.py", line 13, in <module>
    from database import DB
  File "c:\Users\Cybineer\Desktop\Python_Shark_Attack_Analysis\database.py", line 14, in <module>
    from createaccount import createUserAccount
  File "c:\Users\Cybineer\Desktop\Python_Shark_Attack_Analysis\createaccount.py", line 13, in <module>
    from database import DB
ImportError: cannot import name 'DB' from partially initialized module 'database' (most likely due to a circular import) (c:\Users\Cybineer\Desktop\Python_Shark_Attack_Analysis\database.py)

**Comment by AkshayKumar22A** [comment]
You should import and initialize db from createaccount.py then import db to models file
**from flask_sqlalchemy import SQLAlchemy**
Then initialize it
**db = SQLAlchemy(app)**
In models.py
**from app import db**

**Comment by ProgrammingWithCybineer** [comment]
models.py and apps.py??? I dont even have those file in this project. Im kinda confuse as to what you are saying. can you please provide more details. thank you so much

**Comment by AkshayKumar22A** [comment]
With SQLAlchemy installed, we have to import the SQLAlchemy class from the flask_sqlalchemy module into the main application file (__init__.py) or (Main.py) file like this of your project.

# import sqlaclhemy from the flask_sqlachemy module
from flask_sqlalchemy import SQLAlchemy           
or
from app import db
and then you can 
# set the database uniform resource identifier for the database
app.config["SQLALCHEMY_DATABASE_URI"] = 'sqlite:///nameOfDatabase.db'   #in the file in which u want to run the database
With this setup, we have to initialize the database connection by creating an object of the SQLAlchemy class, then we pass our application as a parameter.
# create an instance of the database module
db = SQLAlchemy(app)           #in the file in which u want to run the database


PS - This worked for me but I am new to Github and it will be my first contribution.

**Comment by ProgrammingWithCybineer** [comment]
oh oh ... ok ...now I see. This is my fault I should have closed this issue before. I was able to fix this error. I was importing what i needed incorrectly and in the wrong location. if you look at the recent database file i committed on line 215 and line 262, it works there. all I needed to do was make the variables needed  global then import and use those variables. Probably not the best method but for this case it worked  😉
